docs/sources/tempo/metrics/_index.md

```diff
@@ -0,0 +1,104 @@
+---
+aliases:
+- /docs/tempo/latest/metrics-generator/prometheus-compatibility
+title: Prometheus compatibility
+menuTitle: Prometheus compatibility
+description: Learn about using tempo as a Prometheus data source.
+weight: 550
+keywords:
+  - Prometheus
+  - TraceQL
+---
+
+# Prometheus compatibility
+
+{{< docs/experimental product="Prometheus-compatible API"  >}}
+
+This experimental Prometheus-compatible API provides a way to visualize metrics out of traces using TraceQL.
+
+The API will have native support in a future Grafana release, possibly using an internal API, deprecating the need for
+this compatibility layer.
+
+## Configure a Grafana datasource
+
+Once the `local-blocks` processor is enabled in your `metrics-generator`
+configuration, a new Prometheus data source can be added in Grafana to use the
+`query-frontend` at the `/prom` endpoint for this.  Set the data source URL
+parameter to `http://<HOST>:<PORT>/prom`.  For example:
+
+    ```
+    http://query-frontend.trace.svc.cluster.local:3200/prom
+    ```
+
+<p align="center"><img src="datasource.png" alt="Metrics visualization in Grafana "></p>
+
+The datasource test will fail with a `404` status code, but this is accepted
```

Comment:
Data source should be two words, unless you're referring to something specific that is one word, like a field or UI element
```suggestion
The data source test will fail with a `404` status code, but this is accepted
```